ui/pages/connected-accounts/connected-accounts.component.js

```diff
@@ -79,7 +102,37 @@ export default class ConnectedAccounts extends PureComponent {
           setSelectedAddress={setSelectedAddress}
           shouldRenderListOptions
         />
-      </Popover>
-    );
-  }
+      </Box>
+      {subjectHasSnaps && connectedSnapsMetaData.length > 0 && (
+        <Box>
```

Comment:
Is this `<Box>` with no props required?

Reply:
Nope! Replaced it with fragment now since we need a container to wrap two components